### DISCUSSION TITLE
Is Opening Issue or PR for Documentation Improvement Allowed?

### DISCUSSION THREAD

**Discussion by samiUllah1526** [TanStack/query#5395] [Q&A]
I was reading the latest version's react-query documentation and understood the examples. But still believe the example code can be written in a much more straightforward way. For that, I want to open an issue but the issues page says on only issues for bugs are allowed.

It strange to me that for documentation improvements it is not allowed to open an issue.

**Comment by TkDodo** [comment]
that's what discussions are for. What do you want to change?
